packages/nextjs-mf/src/plugins/NextFederationPlugin/apply-server-plugins.ts

```diff
@@ -21,6 +21,14 @@ export function applyServerPlugins(
 ): void {
   // Import the StreamingTargetPlugin from @module-federation/node
   const { StreamingTargetPlugin } = require('@module-federation/node');
+  const chunkFileName = compiler.options?.output?.chunkFilename;
```

Comment:
temporary - needs better home, ideally in container plugin, but want to find a better solution than hashing